app/scripts/controllers/newOverview.js

```diff
@@ -355,7 +355,7 @@ function OverviewController($scope,
     }
   });
 
-  if (!window.OPENSHIFT_CONSTANTS.DISABLE_OVERVIEW_METRICS) {
+  if (!Constants.DISABLE_OVERVIEW_METRICS) {
```

Comment:
@spadgett FYI, had @benjaminapetersen go ahead and fix this one to use the service